tutorials/community/ubuntu-chromebook.md

```diff
@@ -0,0 +1,105 @@
+---
+id: ubuntu-chromebook
+summary: Learn how to install Ubuntu 16.04 LTS in a chroot on Chromebooks, using the third-party crouton tool.
+categories: community
+tags: guide, chromebook
+difficulty: 3
+status: draft
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2018-01-12
+author: Canonical Web Team <webteam@canonical.com>
+
+---
+
+# Install Ubuntu 16.04 LTS in a chroot on a Chromebook, with crouton
+
+## Overview
+Duration: 2:00
+
+Today we'll be installing Ubuntu on your Chromebook, while preserving your original ChromeOS system.
+
+We will use a third-party script called crouton to install Ubuntu using a chroot, giving Ubuntu its own "pretend" root directory system on your machine. This lets you run ChromeOS and Ubuntu side-by-side, being able to flip between the two on-the-fly.
+
+
+### What you'll learn
+
+- How to put your Chromebook into developer mode
+- How to install Ubuntu in a chroot on it
+- Practical and entertaining uses for Ubuntu on Chromebooks
+
+### What you'll need
+
+- A Chromebook
+- Some basic command-line knowledge
+
+negative
+: **crouton is not part of the Ubuntu project**
+While we love the value that crouton provides, please note that crouton is a third-party script. It is not provided by nor supported by Canonical nor by the Ubuntu project. Use of crouton is at your own risk.
+
+## Putting your Chromebook into Developer Mode
+Duration: 4:00
+
+By default, Chromebooks don't allow us to use chroots out-of-the-box. We will need to put our machine into Developer Mode to grant us this power.
+
+negative
+: **Before Getting Started**
+Placing your device into Developer Mode will wipe all data and user information from it. Since ChromeOS is an online-centric operating system, the vast majority of your data will be stored remotely, but make sure you've backed up everything important that's local to your machine before you begin.
+The use of Developer Mode may void your Chromebook's warranty.
+
+To get to Developer Mode, we need to reboot into Recovery. On most Chromebooks, you do so by turning the device off, then holding down the ESC and Refresh keys while you press the Power button.
+
+Once in this mode, press Ctrl-D. You will be prompted with an opportunity to "turn OS verification OFF". Press Enter to do so.

```

Comment:
For all keyboard shortcuts, we use an inline code style, so they should all be wrapped in backticks like this:

 press \`Ctrl-D\`. You will...

which should render like this:

press `Ctrl-D`. You will...